Server/App/utils/database.py

```diff
@@ -78,7 +89,11 @@ def query(self, query, *params):
         return results
 
     def query_transactionally(self, query, *params):
-        self.begin_transaction()
-        results = self.query(query, *params)
-        self.end_transaction()
+        try:
+            self.begin_transaction()
```

Comment:
Also same thing here, I don't think there is any reason to wrap single queries in transactions